UPGRADING.rst

```diff
@@ -10,6 +10,35 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#4673 Eliminate burner accounts
+===============================
+
+When the operator completes the following steps, they should announce for other
```

Comment:
```suggestion
Operator
~~~~~~~~

Complete the steps in the next section. Then announce on `#team-boardwalk` for other developers to do the same.

Everyone
~~~~~~~~

When notified by the operator, complete the following steps:
```

And eliminate the separate CL item for the announcement.

Speaking of CL: The file name is `UPGRADING.rst`, not `Upgrading.rst`